bindings/python/make_torrent.py

```diff
@@ -19,10 +19,13 @@
 
 parent_input = os.path.split(input)[0]
 
+if os.path.isfile(input):
+	# TODO bind the updated create torrent interface
+	fs.add_files(input)
```

Comment:
why did you add this? the loop below adds the files already. If anything, I think the commented out lines above should be removed.

If it's a matter of testing the ``add_files`` function, perhaps that could be done separately in ``test.py``?

also btw, ``add_files()`` is a free function, not a member of ``file_storage``.